src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/XmlObjectSerializerWriteContextComplexJson.cs

```diff
@@ -361,7 +361,7 @@ internal void VerifyType(DataContract dataContract)
                 knownTypesAddedInCurrentScope = true;
             }
 
-            DataContract knownContract = ResolveDataContractFromKnownTypes(dataContract.StableName.Name, dataContract.StableName.Namespace, null /*memberTypeContract*/);
+            DataContract knownContract = ResolveDataContractFromKnownTypes(dataContract.StableName.Name, dataContract.StableName.Namespace, null /*memberTypeContract*/, null);
             if (knownContract == null || knownContract.UnderlyingType != dataContract.UnderlyingType)
```

Comment:
You can remove this method completely. It's called by CheckIfTypeNeedsVerifcation which isn't called by anything so you can remove that method too.